src/definitions/modules/rating.js

```diff
@@ -542,7 +542,7 @@ $.fn.rating.settings = {
         html = ''
       ;
       while(icon <= maxRating) {
-        html += '<i class="'+iconClass+' icon"></i>';
+        html += '<i class="'+String(iconClass).replace(/"/g,"")+' icon"></i>';
```

Comment:
How about defining a helper?

Reply:
Implemented by fb905d2ec7f615e1c71f986691e6ab1208659669 (identical to dropdown)